homeassistant/components/telnet/switch.py

```diff
@@ -60,11 +62,11 @@ def setup_platform(
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
     """Find and return switches controlled by telnet commands."""
-    devices = config.get(CONF_SWITCHES, {})
+    devices: dict[str, Any] = config.get(CONF_SWITCHES, {})
```

Comment:
CONF_SWITCHES is a required key in the co dig schema.
```suggestion
    devices: dict[str, Any] = config[CONF_SWITCHES]
```

Reply:
Please adjust the config access below when instantiating the entity according to the config schema.